### PR TITLE
Refactor unnecessary advancement listeners

### PR DESCRIPTION
**Pull request by polypixeldev**
- Remove unnecessary advancement listeners
- Move diamond message to diamond advancement reward
- Configure claim stickers url

the advancement broadcasts don't seem to be working, i'll look into that

### CODE REVIEW

File: src/main/resources/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # URL to request form to claim exclusive Hack Club stickers
-claim-stickers-url: "https://example.com/claim-stickers"
+claim-stickers-url: "http://hack.af/mc-stickers"
```
Comment (the456gamer) [inline src/main/resources/config.yml:2]: i think this was set to an example, so the url wasnt public :see_no_evil: 
so if someone runs a copy on their own server (for whatever reason)...

File: src/main/java/com/hackclub/hccore/advancements/DiamondsAdv.java [hunk 2]
```diff
@@ -19,13 +22,28 @@ public class DiamondsAdv extends BaseAdvancement {
...
+    public void giveReward(@NotNull Player player) {
+        player.sendMessage(ChatColor.GREEN
+                + "Congrats, you’ve found your very first diamond! You are now eligible for the exclusive (and limited edition!) Hack Club Minecraft stickers. Head over to "
+                + ChatColor.UNDERLINE + this.plugin.getConfig().getString("claim-stickers-url")
+                + ChatColor.RESET + ChatColor.GREEN + " to claim them!*");
+        player.sendMessage(ChatColor.GRAY + ChatColor.ITALIC.toString()
+                + "*This offer only applies to players who have never received the stickers before. If you have, please do not fill out the form again!");
+        player.sendMessage(ChatColor.GRAY + ChatColor.ITALIC.toString()
+                + "You will only see this message once.");
+    }
```
Comment (the456gamer) [inline src/main/java/com/hackclub/hccore/advancements/DiamondsAdv.java:48]: for new msgs / code can the component api be used instead?
this format is deprecated

File: src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java [hunk 3]
```diff
@@ -171,27 +59,4 @@ public void onPlayerAdvancementDone(final PlayerAdvancementDoneEvent event) {
                 .hoverEvent(descriptionComponent
                     .color(frameSpecificColor))));
     }
```
Comment (the456gamer) [inline src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java:61]: This only works with vanilla (or normal cusotm) advancements im guessing (onPlayerAdvancementDone) is a bukkit event

**Review by the456gamer — CHANGES_REQUESTED** [review]

File: src/main/resources/config.yml (hunk 1 above)
Comment (polypixeldev) [reply src/main/resources/config.yml:2]: oops, i forgot about that

File: src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java (hunk 3 above)
Comment (polypixeldev) [reply src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java:61]: yeah, that's probably it. but I can't find something in the UltimateAdvancementAPI that lets you add a listener for when any custom advancement is granted

**Review by the456gamer — APPROVED** [review]

**Pull request merged**